packages/office-ui-fabric-react/src/components/pickers/BasePicker.types.ts

```diff
@@ -102,27 +115,38 @@ export interface IBasePickerProps<T> extends React.Props<any> {
    * @default undefined
    */
   itemLimit?: number;
+
   /**
    * Function that specifies how arbitrary text entered into the well is handled.
    */
   createGenericItem?: (input: string, ValidationState: ValidationState) => ISuggestionModel<T>;
+  
   /**
    * Aria label for the "X" button in the selected item component.
    * @default ''
    */
   removeButtonAriaLabel?: string;
+  
   /**
    * A callback to process a selection after the user selects something from the picker.
    */
   onItemSelected?: (selectedItem?: T) => T | PromiseLike<T>;
+  
   /**
    * The items that the base picker should currently display as selected. If this is provided then the picker will act as a controlled component.
    */
   selectedItems?: T[];
+  
   /**
    * A callback used to modify the input string.
    */
   onInputChange?: (input: string) => string;
+
+  /**
+   * Whether or not suggestions should reposition based on the screen size.
+   */
+  fixSuggestions?: boolean;
```

Comment:
This makes no sense to me if i'm a customer reading the documentation!

Why on earth would I not want it to reposition? The name itself is confusing. The usage is not clear. The documentation provides no details about defaults or when I would care to deviate...